### PR TITLE
[Needs Review] B07902111 張鈰瀚

### PR DESCRIPTION
**Pull request by cehann**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

**Comment by cehann** [comment]
@amjltc295  Teacher, I have force pushed it, thanks!

**Comment by amjltc295** [comment]
@cehann You should include only one file changed. Also, your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ and https://www.git-tower.com/learn/git/faq/change-author-name-email to fix it.

**Comment by cehann** [comment]
@amjltc295  Teacher, there are two files that I don't need to submit. If I just need to delete it ?

**Comment by amjltc295** [comment]
You have to fix the conflicts when rebasing the code. The latest changes should be kept.

**Comment by amjltc295** [comment]
@cehann Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ and https://www.git-tower.com/learn/git/faq/change-author-name-email to fix it.
By the way, TA means teaching assistant, not teacher ...

File: src/students/B07902111.py [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+    # TODO: download the image from img_url with the request module
+    # and add your student ID on it with draw_name() in the utils module
+    # under src/.
+    from PIL import Image
+    from io import BytesIO
+    import utils
+    r = requests.get(img_url)
+    result_img = Image.open(BytesIO(r.content))
+    result_img = utils.draw_text(Image.open(io.BytesIO(html)), 'b07902111')
```
Comment (amjltc295) [inline src/students/B07902111.py]: Where is html? Did you test your code?

Comment (cehann) [reply src/students/B07902111.py]: @amjltc295  Ok, I revised it!

**Comment by amjltc295** [comment]
Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ and https://www.git-tower.com/learn/git/faq/change-author-name-email to fix it.

**Comment by cehann** [comment]
@amjltc295  I want merge.

**Comment by amjltc295** [comment]
-5 for swearing, -5 for multiple PRs

**Pull request merged**